### ISSUE TITLE
Entry depth of basal melt

### ISSUE THREAD

**Issue by matthew-england-unsw** [pedrocol/basal_mom5-collaborative-project#16]
Test how simulation looks applying basal melt as per experiment BG03 from Mathiot et al.  Run time of ~3-5 years should be ample for a quick look/see.

**Comment by matthew-england-unsw** [comment]
https://www.researchgate.net/publication/318715926/figure/fig1/AS:614357545267225@1523485661537/Freshwater-and-associated-latent-heat-introduced-a-at-the-surface-R-noISF-b_W640.jpg

**Comment by adele157** [comment]
Mathiot et al. found the PAR case with basal melt distributed over depth was better in an idealised case (more similar circulation and temperature anomaly):
<img width="400" alt="Screen Shot 2022-08-11 at 10 54 53 am" src="https://user-images.githubusercontent.com/8506963/184047482-295a2063-71ab-42a6-8014-fc84b3b55178.png">
 What's the dynamical explanation for why BG03 would be better than PAR in the realistic case?

**Comment by matthew-england-unsw** [comment]
My motivation for the quick look at this is more about the BG03 case more closely mimicking actual MW fluxes that would appear adjacent to the ice shelf cavity.   Modulo Ben's comment that it's not always simply a case of the basal melt water floating up to the drafting line.   Second motivation is more around a quick check "does it matter"?   i.e. how important is that choice of the depth range over which we apply the MW.

**Comment by pedrocol** [comment]
I'm about to run this test. Normally the front is located at depth=200m (k=31), I will distribute the basal melt between the front and 50m deeper, which gives a distribution at depth in about 3 vertical levels, which would give a smoother solution if we put all the runoff in one single vertical level

**Comment by adele157** [comment]
Pedro ran 2 years of this experiment. Experiment GPC007 has basal melt input at the ice shelf draft, which can be compared to the first 2 years of GPC008, which is identical, but has a deeper distribution of basal melt input. Try comparing T/S maps at the surface and depth between these runs and compared to obs.

**Comment by adele157** [comment]
That sounds reasonable Matt. Though without the real ice shelf case, there
is the challenge that if they're different, we wouldn't be able to say
which is more correct.

**Comment by matthew-england-unsw** [comment]
Agreed — The idea being if they’re different, we dig deeper. If they’re not different, that’s useful to know.

**Comment by adele157** [comment]
@pedrocol did you ever check this? I seem to remember you running a test simulation, but don't remember seeing the results.
